workflows/pipe-common/scripts/autoscale_sge.py

```diff
@@ -22,6 +22,7 @@
 import time
 import multiprocessing
 import re
+from enum import Enum
```

Comment:
I'm not sure that `enum` package is available in python 2. We can just use the same `CloudProvider` and `AllocationRule` classes with an additional factory methods.

Reply:
It's here, so let it be